db/migrate/20151211163428_add_content_id_to_organisations.rb

```diff
@@ -0,0 +1,13 @@
+class AddContentIdToOrganisations < ActiveRecord::Migration
+  def up
+    add_column :organisations, :content_id, :string
+
+    Organisation.reset_column_information
+
+    ImportOrganisations.new.call
```

Comment:
I have mixed feelings about doing network calls in a migration. The rake task will run anyway at 3am: https://github.gds/gds/alphagov-deployment/blob/master/contacts/admin_to_upload/config/schedule.rb